samples/meetings-notification/nodejs/teamsAppManifest/manifest.json

```diff
@@ -1,43 +1,76 @@
 {
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
   "version": "1.0.0",
-  "id": "<<APP_ID>>",
+  "id": "<<MICROSOFT-APP-ID>>",
   "packageName": "com.microsoft.teams.samples",
   "developer": {
     "name": "Microsoft",
-    "websiteUrl": "https://example.azurewebsites.net",
-    "privacyUrl": "https://example.azurewebsites.net/privacy",
-    "termsOfUseUrl": "https://example.azurewebsites.net/termsofuse"
+    "websiteUrl": "https://microsoft.com",
+    "privacyUrl": "https://microsoft.com/privacy",
+    "termsOfUseUrl": "https://microsoft.com/termsofuse"
   },
   "icons": {
     "color": "icon-color.png",
     "outline": "icon-outline.png"
   },
   "name": {
-    "short": "Content Bubble",
-    "full": "Meeting Extensibility in Microsoft Teams"
+    "short": "Targeted In-Meeting Notification",
+    "full": "Targeted Meeting Notification Demo"
   },
   "description": {
-    "short": "Content Bubble Scenario",
-    "full": "Content Bubble Scenario Test"
+    "short": "Targeted In-Meeting Notification demo",
+    "full": " Sample app to demo how to send in meeting targeted notifications."
```

Comment:
This sample app shows how to send in meeting targeted notifications.

Reply:
Done